src/components/publishing/related_articles/related_article_figure.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react"
+import styled from "styled-components"
+import { crop } from "../../../utils/resizer"
+import { pMedia } from "../../helpers"
+import AuthorDate from "../author_date"
+import { articleHref } from "../constants"
+import Fonts from "../fonts"
+
+interface RelatedArticleFigureProps extends React.HTMLProps<HTMLDivElement> {
+  article: {
+    thumbnail_title: string
+    thumbnail_image: string
+    slug: string
+    contributing_authors: any
+    published_at: string
+  }
+}
+
+const RelatedArticleFigure: React.SFC<RelatedArticleFigureProps> = props => {
+  const { article } = props
+  return (
+    <ArticleFigure href={articleHref(article.slug)}>
+      <ImageTitle>
+        <BlockImage src={crop(article.thumbnail_image, { width: 510, height: 340 })} alt={article.thumbnail_title} />
+        <ArticleTitle>{article.thumbnail_title}</ArticleTitle>
+      </ImageTitle>
+      <AuthorDate authors={article.contributing_authors} date={article.published_at} layout="condensed" />
+    </ArticleFigure>
+  )
+}
+
+const ImageTitle = styled.a`
+  display: flex;
+  flex-direction: column;
+  color: black;
+  text-decoration: none;
+  height: 270px;
```

Comment:
It is possible that long titles could overflow this height?  Maybe min-height would make sense here -- not sure if the character count on thumbnail titles will cover it.